web/twa-vis-platform/code/README.md

```diff
@@ -74,6 +74,13 @@ The uploaded content provided by the deploying developer should match the direct
 * `optional-pages/`: Markdown files for optional static content (with metadata from [gray-matter](https://www.npmjs.com/package/gray-matter)).
 * `style-overrides.css`: Optional CSS overrides.
 
+### 1.6 Reverse Proxy urls
+The default Next.js configuration is designed to function seamlessly when the base URL starts directly from the domain. However, in cases where subdomains are utilised (e.g., http://www.example.org/subdomain/), as seen in stack deployment and reverse proxy scenarios, additional configuration is necessary for Next.js to operate. This includes specifying the `assetPrefix` option in the `next.config.js` file.
```

Comment:
Example here is a page path, rather than a subdomain (which has a [specific meaning](https://www.semrush.com/blog/what-is-a-subdomain/)) :)

Reply:
Thank you for pointing this out :)

I will make the corrections soon